SOC/common/simulation.py

```diff
@@ -216,6 +216,26 @@ def animate(i):
             display(HTML(anim.to_html5_video()))
         else:
             return anim
+
+    def save(self, file_name = 'sim'):
+        """ serialization of object and saving it to file"""
+
+        root = zarr.open_group('state/' + file_name + '.zarr', mode = 'w')
+        values = root.create_dataset('values', shape = (self.L_with_boundary, self.L_with_boundary), chunks = (10, 10), dtype = 'i4')
```

Comment:
A jakbyś tu dodał `data = self.values`, zamiast `shape` i `dtype` (bo jestem pewien że to `zarr` da radę odczytać z danych?

BTW `dtype='i4'` nie przejdzie przy floatowym modelu trzęsień ziemi, tam są floaty.